CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.1.0 | [PR#4218](https://github.com/bbc/psammead/pull/4218) Bump dependencies |
```

Comment:
Do we want to bump to 4.1.0 or 4.0.63 ?

According to https://semver.org/ I would classify these changes as PATCH version bump.

Reply:
Changed in https://github.com/bbc/psammead/pull/4218/commits/de2aa7b802685bf5d7e106a578c5ba86d6b9d54c.